code/game/gamemodes/game_mode.dm

```diff
@@ -179,7 +179,7 @@
 		for(var/mob/living/M in GLOB.mob_living_list)
 			if(!M.mind)
 				continue
-			if(!is_special_type(M, subantag.attached_antag_datum))
+			if(!is_special_type(M, subantag.attatched_antag_datum))

```

Comment:
The spelling was correct before. You made it wrong

Reply:
I am an idiot 🤨